src/accounts/api/views.py

```diff
@@ -181,3 +191,34 @@ def post(self, request):
         public_data = PublicAccountDataSerializer(account).data
 
         return Response(public_data, status=status.HTTP_200_OK)
+
+class ChangePasswordView(GenericAPIView):
+    permission_classes = (AllowAny,)
+    serializer_class = ChangePasswordSerializer
+
+    def get(self, unique_identifier, token):
+        try:
+            account = Account.objects.get(unique_identifier)
+        except (TypeError, ValueError, OverflowError, User.DoesNotExist):
+            return Response({'detail': 'Invalid link or expired.'}, status=status.HTTP_400_BAD_REQUEST)
+
+        if account is not None and default_token_generator.check_token(account, token):
+            return Response({'detail': 'Password reset successfully.'}, status=status.HTTP_200_OK)
+        else:
+            return Response({'detail': 'Invalid link or expired.'}, status=status.HTTP_400_BAD_REQUEST)
+        
+class RequestPasswordResetView(GenericAPIView):
+    permission_classes = (AllowAny,)
+    serializer_class = ChangePasswordRequestSerializer
+
+    def post(self, request):
+        serializer = self.serializer_class(data=request.data)
+        try:
+            serializer.is_valid(raise_exception=True)
+        except ValidationError as e:
+            return Response(data={"error V": str(e)}, status=e.status_code)
+        except Exception as e:
+            return Response(data={"error E": str(e)}, status=status.HTTP_500_INTERNAL_SERVER_ERROR)
+        serializer.create(serializer.validated_data)
```

Comment:
```suggestion
        serializer.save()
```

this will call the create method properly and actually save the new object in the database.